### ISSUE TITLE
Establish the resource of sample DICOMs + target conversions

### ISSUE THREAD

**Issue by yarikoptic** [bids-standard/bids-specification#1516]
IIRC the benefit of having such a resource was initially suggested by @neurolabusc in a conversation with the BIDS Steering group. I have ran the idea by DICOM WG-16 (ref: #1515) who agreed that it might be useful but it would be for us to establish it but it might benefit/relate to ongoing [IHE SHARAZONE](https://www.ihe-europe.net/IHE_SHARAZONE)  (see [a publication providing initial feedback on taking part in it](https://assets.researchsquare.com/files/rs-2832827/v1/d5fec396-7659-4932-bc4d-f02d7f8218a2.pdf?c=1683129013)).

Also I was told that @neurolabusc and @francopestilli (please chime in) are working on https://brainlife.io/ezbids/ and establishing already some collection for its validation etc. May be it could be generalized (for different converters etc) and serve as a beginning for such a community/shared resource.

**Comment by neurolabusc** [comment]
@yarikoptic we have 32 [dcm_qa_*](https://github.com/neurolabusc?tab=repositories&q=dcm_qa&type=&language=&sort=) repositories. These are each designed to exhibit edge cases for conversion, and is useful for regression testing. Hopefully, other tools should provide similar fields for the BIDS tags, but note that [dcm2niix also includes some fields that are not in BIDS](https://github.com/rordenlab/dcm2niix/tree/master/BIDS), so a cross-conversion validation tool would need to have a dictionary of which fields are included in BIDS.

**Comment by yarikoptic** [comment]
True, we would need to accompany with the spec on what fields we expect conversion to have etc. 
Well -- we might want a "golden target" and then let tools to require to match it and then possibly to add extra if they would desire to add more fields (like dcm2niix adds its own version, and we do the same in heudiconv and also extract more metadata from dicoms etc).  Among those 32 do you have any which has a full fledged BIDS dataset(s) as output or just niftis + json sidecars?
Because, I guess, for BIDS conversion tools/targets it would be needed to ensure that we are getting valid BIDS datasets. Well, we might need to do some composition there since often we might want multiple protocols together (e.g., fmap + func) to actually establish some pragmatically useful use case.
